src/main/kotlin/dev/minn/jda/ktx/inject.kt

```diff
@@ -17,14 +17,28 @@
 package dev.minn.jda.ktx
 
 import net.dv8tion.jda.api.JDABuilder
+import net.dv8tion.jda.api.hooks.IEventManager
+import net.dv8tion.jda.api.hooks.InterfacedEventManager
 import net.dv8tion.jda.api.sharding.DefaultShardManagerBuilder
+import java.util.function.IntFunction
+
+
+/**
+ * Applies the [CoroutineEventManager] to this builder.
+ */
+fun JDABuilder.injectKTX(delegate: IEventManager? = null) =
```

Comment:
Why not just set this to the actual default instead of null? This makes it harder to understand for the library user

Reply:
There is no explanation what "delegate" is here. Please add some documentation since this is non-obvious.

Reply:
will do.